### DISCUSSION TITLE
Rename export to download

### DISCUSSION THREAD

**Discussion by joshnuss** [SvelteLab/SvelteLab#258] [Ideas]
Hi!

Would you consider re-labelling "Export" to "Download" or "Download project"?
I think it will be easier to find the option, since in the mind of the user that is their intent

**Comment by paoloricciuti** [comment]
I think we can add a seo tag for this but I'm not opposed to rename to...wdyt @SarcevicAntonio ?

**Comment by SarcevicAntonio** [comment]
Sure 😁👍

Paolo Ricciuti ***@***.***> schrieb am Mi., 19. Juli 2023,
03:05:

**Comment by joshnuss** [comment]
Shall I open a PR?
I believe it's just a one-line [change here](https://github.com/SvelteLab/SvelteLab/blob/6e9235f821410fe0699afa1467543d89e8e8cb00/src/lib/command_runner/commands.ts#L259C15-L259C15)

**Comment by paoloricciuti** [comment]
> Shall I open a PR? I believe it's just a one-line [change here](https://github.com/SvelteLab/SvelteLab/blob/6e9235f821410fe0699afa1467543d89e8e8cb00/src/lib/command_runner/commands.ts#L259C15-L259C15)

Sure, feel free to do so...I was planning to do it but I had literally 0 time yesterday...if you don't do it I'll do it today;)
